### ISSUE TITLE
Default build processes fail

### ISSUE THREAD

**Issue by Chandler-Davidson** [hsv-dot-beer/vue-hsvdotbeer#37]
I tried following the readme by running `npm run dev` and `npm run build`, but both fail to compile. I got the following error for both: 
```
Module parse failed: Unexpected token (755:8)
You may need an appropriate loader to handle this file type.
```

It seems that the webpack config is [missing the vue-loader plugin](https://vue-loader.vuejs.org/guide/#manual-setup). Am I missing information and need to update the docs? Or do I need to fix the build process?

**Comment by mjcarroll** [comment]
Yeah, stuffs broken.  I haven't had time to really look at this lately, but it's on my list...

**Comment by Chandler-Davidson** [comment]
@mjcarroll If you know what's wrong, I don't mind updating docs. Otherwise, I'll poke around and see where I can fix it.
